MapboxCoreNavigation/NavigationService.swift

```diff
@@ -505,7 +505,7 @@ extension MapboxNavigationService {
     }
     
     public var location: CLLocation? {
-        return self.locationManager.location
+        return self.router.location
```

Comment:
If I’m not mistaken, this is a change from raw locations to snapped locations. Are there implications beyond location-based events? If so, we should note the change in the changelog.

Reply:
Good point. We do want to use raw locations for events. We don't use `NavigationService.location` in any other places in the SDK so it shouldn't have any other implications. Would you mind re-reviewing https://github.com/mapbox/mapbox-navigation-ios/pull/1994/commits/e060c8d905c02f8f867d4ed4c4098ba959ad0046?